app/models/repository_rows_my_module.rb

```diff
@@ -0,0 +1,7 @@
+class RepositoryRowsMyModule < ActiveRecord::Base
```

Comment:
I don't know. Maybe we messed up with the rest of the join tables, which are all named e.g. `user_my_modules`, but that translates into ActiveRecord being named `UserMyModule`, which I believe is a good way to name a join model. I mean, I think it's more consistent to keep this table named `repository_row_my_modules`, which will then also mean that the ActiveRecord is named `RepositoryRowMyModule` (with no plural there).